utils/index.js

```diff
@@ -192,6 +192,16 @@ const currencyFilter = (number, fractionDigits = 2) => {
   })
 }
 
+const sortByLineNumber = (...objToSort) => {
```

Comment:
It would be nice to have 2-3 tests for this but given ⏲ ⌛️ ⌚️, we can do them l8r.